doc/api/async_context.md

```diff
@@ -116,17 +116,35 @@ Each instance of `AsyncLocalStorage` maintains an independent storage context.
 Multiple instances can safely exist simultaneously without risk of interfering
 with each other's data.
 
-### `new AsyncLocalStorage()`
+### `new AsyncLocalStorage([options])`
 
 <!-- YAML
 added:
  - v13.10.0
  - v12.17.0
+changes:
+ - version: REPLACEME
+   pr-url: https://github.com/nodejs/node/pull/45386
+   description: Add option onPropagateCb.
 -->
 
+> Stability: 1 - `options.onPropagateCb` is experimental.
+
+* `options` {Object}
+  * `onPropagateCb` {Function} Optional callback invoked before a store is
```

Comment:
I would drop the `Cb` suffix as it's implied by the `on` prefix that this will run when a future thing happens. It's also not a particularly descriptive fragment being `Cb` rather than a more readable `Callback`, which makes it not particular friendly cross-language.

Reply:
done